docs/UserGuide.md

```diff
@@ -278,3 +319,10 @@ If your changes to the data file makes its format invalid, LetsGetHired will dis
 | **Help**   | `help`                                                      | `help`                                                                |
 | **Clear**  | `clear`                                                     | `clear`                                                               |
 | **Exit**   | `exit`                                                      | `exit`                                                                |
```

Comment:
Perhaps we could include the `find` feature here as well in the command summary?